### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by xuhyun11** [totoroyun/early-stage-phishing#1]
It is written in your paper and code that only the data before T-W is contained in a time window. Then how are the node features extracted from the verification set and test set? There is no transaction information in the validation set and test set time in GT-W?

**Comment by YunyunWan** [comment]
Hi,


Sorry for the delayed response. Your understanding is spot on. We do indeed gather node data for the training, validation, and test sets at time t-w. These nodes have engaged in transactions at least before t-w, ensuring their presence in the transaction network at that time, which helps with feature extraction. Since our main focus is on identifying early-stage phishing users, those in the validation and test sets have all stopped their fraudulent activities after time t-w. Regarding the statement "There is no transaction information in the validation set and test set time in GT-W?" I'm not entirely sure what it means.


Best,
Yun



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "totoroyun/early-stage-phishing"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年1月14日(星期天) 下午4:43
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[totoroyun/early-stage-phishing] dataset (Issue #1)





 
It is written in your paper and code that only the data before T-W is contained in a time window. Then how are the node features extracted from the verification set and test set? There is no transaction information in the validation set and test set time in GT-W?

**Comment by xuhyun11** [comment]
Thank you for your answer. I think I understand what you mean.
Best wishes for you.
